homeassistant/components/airzone_cloud/sensor.py

```diff
@@ -129,6 +167,27 @@ def __init__(
         self._async_update_attrs()
 
 
+class AirzoneWebServerSensor(AirzoneWebServerEntity, AirzoneSensor):
+    """Define an Airzone Cloud WebServer sensor."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        description: SensorEntityDescription,
+        entry: ConfigEntry,
+        ws_id: str,
+        ws_data: dict[str, Any],
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator, entry, ws_id, ws_data)
+
+        self._attr_name = f"{ws_id} {description.name}"
```

Comment:
You could consider implementing `has_entity_name`, removing the need to concat the name (and use just the description name instead)

Reply:
But in that case, what would happen if the user had multiple WebServers?
How could the user differentiate them in that case?

Reply:
The device name would take care of that case, in this case, the entity name is prepended. to resolve the issue, while in the new naming style, the device should take care of that (while the entities only got names based on what they are themselves, without taking the device into account).

Reply:
Done in 64f75e3c9f714148841cbf727f5ee48e14fb2485